schemas/2017-03-01-preview/Microsoft.Sql.json

```diff
@@ -326,36 +326,6 @@
       ],
       "description": "Microsoft.Sql/servers/databases/extendedAuditingSettings"
     },
-    "servers_databases_vulnerabilityAssessments": {
```

Comment:
>servers_databases_vulnerabilityAssessments [](start = 5, length = 42)

this is a breaking change. Is the API already removed from service?

Reply:
No, and it won't.
This is a temporary fix for a problem we are working on.